console/console-init/ui/src/modules/address-space/components/AddressSpaceReview/AddressSpaceReview.tsx

```diff
@@ -99,7 +131,7 @@ export const AddressSpaceReview: React.FC<IAddressSpaceReviewProps> = ({
             )}
             {plan && plan.trim() !== "" && (
               <>
-                <GridItem span={5} style={{ marginBottom: 16, marginRight: 5 }}>
+                <GridItem span={5} className={css(style.preview_info_gridItem)}>
```

Comment:
This component here is repetitive and we can create a generic component and just pass Namespace, plan,type as props/arguments.